### ISSUE TITLE
Coinomi wallet not visible under Android wallets

### ISSUE THREAD

**Issue by GeorgeKimionis** [bitcoin-dot-org/bitcoin.org#1569]
According to [the page](https://bitcoin.org/en/choose-your-wallet)'s source Coinomi should be visible under the Android section. Is there an issue with the web page?

Linking issue: https://github.com/Coinomi/coinomi-android/issues/146

**Comment by harding** [comment]
The layout is set to only show two rows of wallets (12 total) but there are currently 14 wallets in the Android section, so the final two are `display: none`.  Here's the page with the final two shown:

![2017-04-21-23_39_57_445187377](https://cloud.githubusercontent.com/assets/61096/25301076/1384dd84-26ec-11e7-97aa-0aedbb465063.png)

The ordering of the wallets should be randomized each day via JS (within certain [general groupings](https://github.com/bitcoin-dot-org/bitcoin.org/pull/473)), so it will likely be shown again tomorrow.

Ultimately, I think someone will either have to update the CSS to work with more than two rows of wallets or we'll have to remove some wallets.  I know we've discussed this before, but I can't immediately find the issue or PR where we discussed it.
